Content.Client/_DV/Surgery/SurgeryDirtinessItemStatus.cs

```diff
@@ -27,19 +42,60 @@ protected override void FrameUpdate(FrameEventArgs args)
         if (!_entMan.TryGetComponent<SurgeryDirtinessComponent>(_uid, out var comp))
             return;
 
-        if (_dirtiness == comp.Dirtiness)
+        var isTool = _entMan.HasComponent<SurgeryToolComponent>(_uid);
+
+        if (_dirtiness == comp.Dirtiness && !isTool)
             return;
         _dirtiness = comp.Dirtiness;
 
-        Clear();
+        if (isTool && _container.TryGetContainingContainer((_uid, null, null), out var container))
+        {
+            var user = container.Owner;
+
+            FixedPoint2? glovesDirtiness =
+                _inventory.TryGetSlotEntity(user, "gloves", out var gloves)
+                    ? _entMan.TryGetComponent<SurgeryDirtinessComponent>(_uid, out var glovesComp)
+                        ? glovesComp.Dirtiness
+                        : FixedPoint2.Zero
+                    : null;
```

Comment:
this should be an inventory relayed event for gloves and mask slots (mask isnt even checked in this)

Reply:
why though, that's not how the surgery system checks for gloves

Reply:
it should

Reply:
well it doesn't and this is a PR that improves the UI slightly not a refactor shitmed to make it less bad PR